WcaOnRails/db/structure.sql

```diff
@@ -2,349 +2,20 @@
 /*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
```

Comment:
Oh my, the diff in this file is enormous. I'd guess that your `mysqldump` is behaving differently than other people's? Any idea why this is?

Reply:
I started looking into this but didn't have much luck. :/
I just messaged you on Slack to try to debug this further.